docs/src/getting-started/installation.rst

```diff
@@ -0,0 +1,98 @@
+.. meta::
+  :description: Tensile documentation and API reference
+  :keywords: Tensile, GEMM, Tensor, ROCm, API, Documentation
+
+.. _installation:
+
+********************************************************************
+Installation
+********************************************************************
+
+You can obtain Tensile as part of ROCm installation or choose to install Tensile from source. This document discusses both the methods to obtain Tensile.
+
+Install ROCm
+============
+
+1. Install ROCm for your platform. For installation instructions, refer to `Linux <https://rocm.docs.amd.com/projects/install-on-linux/en/latest/tutorial/quick-start.html>`_ or Windows `Windows <https://rocm.docs.amd.com/projects/install-on-windows/en/latest/index.html>` installation guide.
+
+   After the installation is complete, you can find the binaries and libraries in ``/opt/rocm``. Installing ROCm provides compilers such as ``amdclang++``, and other useful tools including ``rocminfo`` and ``rocprofv2``.
+
+   .. tip::
+
+      If using Bash, we recommend you to set ``PATH=/opt/rocm/bin/:$PATH`` in your ``~/.bashrc`` and refresh your shell, using ``source ~/.bashrc``.
+      Alternatively, export the path for your current shell session only, using ``export PATH=/opt/rocm/bin/:$PATH``.
+
+Install OS dependencies
+=======================
+
+Here are the steps to install dependencies for Ubuntu distributions. For other distributions, use the appropriate package manager.
```

Comment:
@bstefanuk For other distributions, where can the users find the appropriate package manager? Can we provide ROCm package manager link here?

Reply:
This would be a significant addition that probably won't add much value. Typically those who work on non-Ubuntu OS's such as RHEL will be familiar with the package manager of their OS. The following page (linked in the docs in step 1) show the package managers for the different OSs used when installing ROCm. That could be used as a reference.

Reply:
Ok. Then, let's leave it like that.